src/components/ImagesCarousel.tsx

```diff
@@ -27,7 +24,10 @@ const images = [
 ];
 
 const ImagesCarousel = () => {
-
+  const [visible, setModalVisible] = useState(false);
+  const [modalImage, setModalImage] = useState("");
+  const [modalImageIndex, setModalImageIndex] = useState(0);
+  
   return(
   <View>
     <Text style={styles.headingText}>IMAGES</Text>
```

Comment:
maybe change this to a map function so that there can be a varying number of images on the carousel

Reply:
A good idea. @cam-arch, what do you think?